text-inputs/README.md

```diff
@@ -6,253 +6,144 @@ You can find the design specs on [www.decathlon.design](https://www.decathlon.de
 
 ## Usage with Vitamin Theme
 
-By inheriting the Vitamin theme in your app, you just have to use the default `TextInputLayout` component or the `VitaminTextInputLayout*` component.
+By inheriting the Vitamin theme in your app, you just have to use the default `TextInputLayout`
+component or the `VitaminTextInputLayout*` component.
+
+> **Disclaimer**
+In this file, `Theme.Vitamin.<ThemeName>` will be used to indicate the different Vitamin themes. You
+can retrieve [the Vitamin themes list here](../vitamin/README.md).
 
 ```kotlin
 implementation("com.decathlon.vitamin:vitamin:<version>")
 ```
 
+```xml
+
+<style name="AppTheme" parent="Theme.Vitamin.<ThemeName>" />
+```
+
 ### Filled
 
 ```xml
-<com.decathlon.vitamin.textinputs.VitaminTextInputLayoutFilled
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    app:helperText="Helper text">
 
-    <com.google.android.material.textfield.TextInputEditText
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:hint="hint" />
+<com.decathlon.vitamin.textinputs.VitaminTextInputLayoutFilled android:layout_width="match_parent"
+    android:layout_height="wrap_content" app:helperText="Helper text">
+
+    <com.google.android.material.textfield.TextInputEditText android:layout_width="match_parent"
+        android:layout_height="wrap_content" android:hint="hint" />
```

Comment:
Why do you change the formatting of these elements? (and below)

Reply:
bad habit of shortcut to reformat